<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Bluetooth Advertising Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" >
    <item row="0" column="0" colspan="5" >
     <widget class="QTableWidget" name="activeCampaigns" >
      <property name="cursor" stdset="0" >
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="editTriggers" >
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode" >
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior" >
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <column>
       <property name="text" >
        <string>Identificador</string>
       </property>
      </column>
      <column>
       <property name="text" >
        <string>Nombre</string>
       </property>
      </column>
      <column>
       <property name="text" >
        <string>Fecha inicio</string>
       </property>
      </column>
      <column>
       <property name="text" >
        <string>Fecha fin</string>
       </property>
      </column>
      <column>
       <property name="text" >
        <string>Habilitar</string>
       </property>
      </column>
      <column>
       <property name="text" >
        <string>Reenviar</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="1" column="0" >
     <widget class="QPushButton" name="addCampaign" >
      <property name="toolTip" >
       <string>A単adir una campa単a</string>
      </property>
      <property name="text" >
       <string>+</string>
      </property>
      <property name="shortcut" >
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" >
     <widget class="QPushButton" name="removeCampaign" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="toolTip" >
       <string>Eliminar la campa単a seleccionada</string>
      </property>
      <property name="text" >
       <string>-</string>
      </property>
      <property name="shortcut" >
       <string>-</string>
      </property>
      <property name="flat" >
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="2" >
     <widget class="QPushButton" name="editCampaign" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="toolTip" >
       <string>Propiedades de la campa単a seleccionada</string>
      </property>
      <property name="text" >
       <string>Propiedades</string>
      </property>
      <property name="shortcut" >
       <string>*</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3" >
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>271</width>
        <height>27</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="4" >
     <widget class="QPushButton" name="logButton" >
      <property name="toolTip" >
       <string>Visualizar log de envios</string>
      </property>
      <property name="text" >
       <string>Log</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar" />
 </widget>
 <resources/>
 <connections/>
</ui>
